src/main/java/io/appform/http/client/HttpClient.java

```diff
@@ -27,38 +35,40 @@
     public final OkHttpClient client;
 
     @SneakyThrows
-    public Response post(String url,
-                         final Object payload,
-                         final Map<String, String> headers) {
-        final HttpUrl httpUrl = HttpUrl.get(url);
-        Request.Builder postBuilder;
-        if(payload instanceof String) {
-            postBuilder =  new Request.Builder()
+    public Response post(
+            String url,
+            final Object payload,
+            final Map<String, String> headers) {
+        val httpUrl = HttpUrl.get(url);
+        val postBuilder = new Request.Builder();
+        if (payload instanceof String) {
+            postBuilder
                     .url(httpUrl)
-                    .post(RequestBody.create(APPLICATION_JSON, (String)payload));
+                    .post(RequestBody.create((String)payload, APPLICATION_JSON));
         }
         else {
-            postBuilder = new Request.Builder()
+            postBuilder
                     .url(httpUrl)
-                    .post(RequestBody.create(APPLICATION_JSON, mapper.writeValueAsBytes(payload)));
+                    .post(RequestBody.create(mapper.writeValueAsBytes(payload), APPLICATION_JSON));
```

Comment:
Is the reordering correct ?
https://square.github.io/okhttp/3.x/okhttp/okhttp3/RequestBody.html
In the the order is different.